SwiftFormat/SwiftParser.py

```diff
@@ -1,9 +1,40 @@
 __author__ = 'kronenthaler'
 
+from SwiftTokenizer import SwiftTokenizer
+from SwiftNodeTypes import *
+
 class SwiftParser:
     # the class needs a tokenizer
     # the class needs to store the parsing tree, that will be input to the formatter
 
-    def __init__(self, src):
+    def __init__(self, src=None):
         pass
 
+    def comment(self, tokenizer):
+        comment = self.single_line_comment(tokenizer)
+        if comment:
+            return comment
+        return self.multi_line_comment(tokenizer)
+
+    def single_line_comment(self, tokenizer):
+        head = tokenizer.search_for(u"//")
+        if not head:
+            return tokenizer.push_back()
+
+        comment = tokenizer.next_token(delimiters="\x0A\x0D")
```

Comment:
consider end of string or EOF as terminator: single line comment at the end of the file